src/wasm.h

```diff
@@ -1610,6 +1614,14 @@ class Module {
     }
     importsMap.erase(name);
   }
+
+  void removeImportsWithSubstring(Name name) {
+    for (int i = imports.size() - 1; i >= 0; i--) {
```

Comment:
Except I want to include zero. We need to hit every index in reverse order.
Could also do
```c++
for (size_t i = imports.size(); i > 0; --i) {
  int index = i - 1;
```

Could also create a vector of indices to remove and do this two-pass.
Could also use `std::remove_if` and a lambda, which more directly expresses what we mean.

In any case, looking at this again I should maintain what I assume is a class invariant, suggested by `removeImport` just above, and also erase the name from `importsMap`